src/components/VideoLoader.js

```diff
@@ -0,0 +1,7 @@
+import { VideoCardSkeleton } from "./Skeleton";
+
+export const VideoLoader = () => {
+  return [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12].map((item) => (
```

Comment:
instead of defining an array explicitly, you can use the `Array()` method and generate a prefilled array based on length

`[...Array(13).keys()].splice(1)` will generate a prefilled array of 1 to 12